components/library/libepubgen/Makefile

```diff
@@ -23,18 +23,19 @@
 BUILD_BITS=32_and_64
 include ../../../make-rules/shared-macros.mk
 
```

Comment:
checking for CPPUNIT... no
configure: error: Package requirements (cppunit) were not met:

No package 'cppunit' found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Reply:
Missing dependency?

Reply:
Yes, I just reported it and didn't try to fix it to get more commits done.

Reply:
No problem, just to make sure. It is good for me like that: you tell me what to fix so I focus on the fixing part.